content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Updating Configuration via a Configmap
+content_type: tutorial
+weight: 20
+---
+
+<!-- overview -->
+This page provides a step-by-step example of updating configuration within a Pod via a ConfigMap and builds upon the [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) task. At the end of this tutorial, you will understand how to change the configuration for a running application. 
+This tutorial uses the NGINX image as an example.
+
+## {{% heading "prerequisites" %}}
+{{< include "task-tutorial-prereqs.md" >}}
+
```

Comment:
Please mention that you need `curl` (it's really popular, but even so, let's be clear about what people need to have).

Reply:
Ack. I'm adding it as a new page under the `includes` directory to make it reusable. Any tutorial that needs `cURL` as a prerequisite can include this page in their tutorial to save time and have a standard pattern across tutorials.